### PR TITLE
Add case for issue 2736

### PR DESCRIPTION
**Pull request by maryvictol**
This PR add testcase and e2e-test for Forcible terminate instances if the job is stuck in umount #2736

### CODE REVIEW

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2736.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+| 4 | At the tool page, hover over the "**Run v**" button | | 
+| 5 | Click the "**Custom settings**" button in the list | |
+| 6 | Expand the **ADVANCED** section | |
+| 7 | Input `echo "#!/usr/bin/env bash" > /usr/local/sbin/umount && echo "sleep infinity" >> /usr/local/sbin/umount && chmod +x /usr/local/sbin/umount` command into the ***Cmd template*** field | |
+| 8 | Launch tool | |
+| 9 | At the **Runs** page, click the just-launched run | | 
+| 10 | Open run's *Console* | |
+| 11 | Wait until `Unmounting all storage mounts` appears in the log | |
+| 12 | Wait `CP_TERMINATE_RUN_ON_CLEANUP_TIMEOUT_MIN` default time (1min) | Run isn't stopped automatically |
+| 13 | Repeat steps 3-7 | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2736.md]: Fix to `steps 2-7`

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2736.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+| 14 | Hover over **v** button next to "Add parameter" | |
+| 15 | Click on ***String parameter*** item | |
+| 16 | Input `CP_TERMINATE_RUN_ON_CLEANUP_TIMEOUT_MIN` parameter Name and `2` as parameter Value | |
+| 17 | Repeat steps 8-11 | |
+| 18 | Wait until ***CleanupEnvironment*** task appears | |
+| 19 | Click ***CleanupEnvironment*** task | `Will wait for 2min to let the run stop normally. Otherwise it will be terminated.` is shown in the log |
+| 20 | Wait `CP_TERMINATE_RUN_ON_CLEANUP_TIMEOUT_MIN` time specified at the step 16 | <li> Run is stopped <li> `Run <runID> is still running after 2min. Terminating a node.` is shown in the log, where `<runID>` is ID run launched at step 17 |
+
+**After:**
+- Stop the runs launched at steps 9 and 17
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2736.md]: Fix to `Stop the run launched at step 8`.
As for the run launched at step 17 - it was stopped at step 20, isn't it?

**Review by NShaforostov — APPROVED** [review]
Fix minor notes before the merging, please (see comments for details).

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java [hunk 3]
```diff
@@ -35,6 +35,8 @@
 import java.util.stream.Stream;
 
 import static com.codeborne.selenide.Selenide.open;
+import static com.epam.pipeline.autotests.ao.LogAO.*;
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java]: Do we actually need this import?

**Review by kamyshova — APPROVED** [review]

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2736.md (hunk 1 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2736.md]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2736.md (hunk 2 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2736.md]: Done

**Pull request merged**